src/messages.ts

```diff
@@ -17,16 +17,27 @@ import {Release} from './releases';
 const ISSUES_URL =
   'https://github.com/gradle-update/update-gradle-wrapper-action/issues';
 
+const TARGET_VERSION_PLACEHOLDER = '%targetVersion%';
+const SOURCE_VERSION_PLACEHOLDER = '%sourceVersion%';
+
+export function pullRequestTitle(
+  template: string,
+  sourceVersion: string | undefined,
```

Comment:
I noticed in #679 that this could not be undefined; I'd expect both to be similar, no? Should undefined be allowed, or no? Asking, because the `sourceVersion` replace on line 30 will yield a different result for PR title vs commit message.

Reply:
I can't remember which are the cases where `sourceVersion` can't be determined, but it might be worth documenting in the input description that it might happen.

Reply:
Good catch @ChrisCarini, thank you!

In that case it seems better to leave this PR as it is, with `sourceVersion` allowing undefined values.

01d6e8a documents the fallback behaviour in the `README.md` and `action.yml`, and #679 will need to be adjusted to match that same behaviour.